dev/metrics/README.md

```diff
@@ -0,0 +1,31 @@
+# Readme
+This directory contains some docker files to help you set up metrics tracking on an alluxio cluster. The scripts uses opentelemetry's auto instrumentation java agent to generate tracing information for GRPC and S3 calls, which is visualized using Jaeger and Prometheus. 
+
+1. On one of the Alluxio master or the node where you want to run Jaeger and Prometheus, run 
+```
+docker-compose run -e MASTER_IP=xxx.xx.xx.xx -d
```

Comment:
to match the same pattern as the worker, rename the master's docker compose file to docker-compose-master.yaml and set the `-f` flag here?